xknx/telegram/telegram.py

```diff
@@ -15,6 +15,8 @@
 """
 from __future__ import annotations
 
+from datetime import datetime
```

Comment:
we use `time.time()` in other modules when dealing with time. Any specific reason for using datetime here?